concert/loghandler.py

```diff
@@ -67,23 +46,28 @@ class RemoteLoggingHandler(logging.Handler):
     """
 
     _device: RemoteDirectoryWalkerTangoDevice
+    _logger_id: str
     _tasks: List[asyncio.Task]
     _loop: asyncio.AbstractEventLoop
 
     def __init__(
             self,
             device: RemoteDirectoryWalkerTangoDevice,
+            logger_id: str,
```

Comment:
Shouldn't the `logger_id` be hidden in this class? That means it itself registers it, why should anyone else (RemoteDirectoryWalker) do this for it? Seems like a leak from encapsulation.